components/registry/params.go

```diff
@@ -6,7 +6,8 @@ import (
 
 type ParametersRegistries struct {
 	Chains struct {
-		FilePath string `default:"waspdb/chains/chain_registry.json" usage:"the path to the chain registry file"`
+		FilePath         string `default:"waspdb/chains/chain_registry.json" usage:"the path to the chain registry file"`
+		ReadOnlyFilePath string `default:"" usage:"the path to the chain registry file for a read only DB"`
```

Comment:
why are these two different paths? Isn't it simpler to have a single path and a bool?

```suggestion
		FilePath         string `default:"waspdb/chains/chain_registry.json" usage:"the path to the chain registry file"`
		ReadOnly         bool   `...`
```

Reply:
@lmoe which one do you prefer?

Reply:
I would prefer a single argument: "--readOnlyDB" (Similar to how you would pass the config path with "--config").

I would expect to simply provide the `/path/to/waspdb/chains/` path there. 

In `waspdb/chains/` you have 
- chain_registry.json
- data/<chainId>
- index/<chainId>

I think the approach of using the config variables will make things a little confusing. The readonly feature is really only for debugging purposes and I wouldn't include that in the configuration itself. I would handle this with an actual App argument and pass the behavior down to the components.

I basically just want to launch `./wasp --readOnlyDB /path/to/waspdb/chains` and make it start a readonly environment. 

In some cases it can be, that we don't have a chain_registry.json. In this case I would simply create one upon launch into the provided folder.

Reply:
Our CLI is built on github.com/iotaledger/hive.go/app. Introducing a new top-level --readOnlyDB flag would require changes that risk breaking existing behavior or a dedicated parsing path just for that flag. So I previously introduced a scoped flag instead: --registries.chains.readOnlyFilePath=/Users/hauyang/Work/wasp/chains/chain_registry.json.